app/models/metadata-record.ts

```diff
@@ -1,17 +1,53 @@
+import { inject as service } from '@ember/service';
 import Model, { AsyncHasMany, attr, hasMany } from '@ember-data/model';
+import IntlService from 'ember-intl/services/intl';
 
 export interface LanguageText {
     '@language': string;
     '@value': string;
 }
 
 export default class MetadataRecordModel extends Model {
+    @service intl!: IntlService;
+
     @attr('array') resourceType!: string[];
     @attr('array') resourceIdentifier!: string[];
     @attr('object') resourceMetadata!: any;
 
     @hasMany('metadata-record', { inverse: null })
     relatedRecordSet!: AsyncHasMany<MetadataRecordModel> & MetadataRecordModel[];
+
+    get label(): string {
```

Comment:
I think I will use a helper called `{{locale-string}}` or something like that, which will take the `resourceMetadata` object and a key to fetch the language-appropriate value. That should help dry up some of this by just doing `{{locale-string metadataRecord.resourceMetadata 'label'}}`